packages/protocol/contracts/stability/Exchange.sol

```diff
@@ -313,6 +340,20 @@ contract Exchange is
     emit ReserveFractionSet(newReserveFraction);
   }
 
+  /**
+    * @notice Allows owner to set the minSupplyForStableBucketCap,
+    *  the minimal stableToken supply considered for the stableToken cap
+    * @param newMinSupplyForStableBucketCap The new value for the minSupplyForStableBucketCap
+    */
+  function setMinSupplyForStableBucketCap(uint256 newMinSupplyForStableBucketCap) public onlyOwner {
+    minSupplyForStableBucketCap = newMinSupplyForStableBucketCap;
+    require(
+      newMinSupplyForStableBucketCap > 0,
+      "min supply for stable bucket cap must be greather than 0"
```

Comment:
Given the intent of this variable - to combat the tank size being too low if it relies only on the supply - I'm wondering if it makes sense to also hardcode a lower limit for what it can be set. Cose effectively this could be set to `1` effectively making it redundant. Might be overkill but just a thought cc @martinvol